library/build.gradle

```diff
@@ -10,7 +10,7 @@ android {
     compileSdkVersion 25
 
     defaultConfig {
-        minSdkVersion 14
+        minSdkVersion 21
```

Comment:
PendingIntent.FLAG_IMMUTABLE is added in SDK 21 (Android 5.0), so this bump is required.